pandas/conftest.py

```diff
@@ -881,3 +881,72 @@ def index_or_series(request):
     See GH#29725
     """
     return request.param
+
+
+_indexes = {
+    "bool-index": tm.makeBoolIndex(10, name="a"),
+    "int-index": tm.makeIntIndex(10, name="a"),
+    "float-index": tm.makeFloatIndex(10, name="a"),
+    "date-index": tm.makeDateIndex(10, name="a"),
+    "localized-date-index": tm.makeDateIndex(10, name="a", tz="US/Eastern"),
+    "period-index": tm.makePeriodIndex(10, name="a"),
+    "string-index": tm.makeStringIndex(10, name="a"),
+    "unicode-index": tm.makeUnicodeIndex(10, name="a"),
+}
+
+
+@pytest.fixture(params=_indexes.keys())
+def index(request):
+    """
+    Fixture for tests on indexes
+
+    copy to avoid mutation, e.g. setting .name
```

Comment:
for now just create fixtures in pandas/tests/base

then can move common fixtures to a higher level. *only* really shared fixtures should be in this file.